python/ray/_private/services.py

```diff
@@ -1675,6 +1675,14 @@ def determine_plasma_store_config(object_store_memory,
             # /dev/shm.
             if shm_avail > object_store_memory:
                 plasma_directory = "/dev/shm"
+            elif not os.environ.get("RAY_OBJECT_STORE_ALLOW_SLOW_STORAGE"):
+                raise ValueError(
```

Comment:
I am concerned that this error might not get raised in cases where:
1. `object_store_memory` is capped to `get_shared_memory_bytes()`
2. Some small amount of `/dev/shm` is freed
3. `shm_avail` is now slightly larger than  `object_store_memory` (but substantially lower than `avail_memory * DEFAULT_OBJECT_STORE_MEMORY_PROPORTION`)

And the only sign of this would be the `debug` log of the object store being capped.

Reply:
That's the intended behavior right? We auto-cap and only log a warning.

If you manually set it, there is no auto capping and this will always be raised.